docker/Dockerfile_centos

```diff
@@ -0,0 +1,30 @@
+FROM centos
```

Comment:
If these are working dockerfiles, it should be pretty simple to create a Jenkins job to test these, but maybe only do this manually, as it may be a bit hefty to run this with every commit. Recommendations @amues ?